packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -4370,11 +4379,9 @@ export class ContainerRuntime
 		batchId: BatchId,
 		staged: boolean,
 	): void {
-		this.orderSequentially(() => {
```

Comment:
i don't think this diff anything, and it created reentrancy problems. the fact that it is synchronous will have it land in a batch either regardless

Reply:
What about if we're in ImmediateMode?

Reply:
isn't that deprecated? and it looks unused

Reply:
additionally, if you look at order sequentially before my change, it doesn't do anything in the non-rollback case, it just executes synchronously, which is the same as here.

Reply:
Anyone on Declarative Model running in Compat Mode "1" (to allow collab with 1.x clients) has it enabled.

Reply:
> it doesn't do anything in the non-rollback case

It increments `_orderSequentiallyCalls` which has usages throughout CR.  Most importantly, see `currentlyBatching` and its callsite in `submit`.  Without calling `orderSequentially` here, every op will be flushed independently during resubmit.

Reply:
i still don't think it matter. order sequentially doesn't do anything here

Reply:
ahh. so it's wholly based on side effects that just happen to align with being inside order sequentially